SpywareFilter/sections/cname_trackers.txt

```diff
@@ -44,6 +44,9 @@
 ! Eulerian : eulerian.net disguised trackers
 !#include https://raw.githubusercontent.com/AdguardTeam/cname-trackers/master/trackers/eulerian.txt
 !
+! Extole : extole.com disguised trackers
+!#include https://raw.githubusercontent.com/AdguardTeam/cname-trackers/master/trackers/extole.txt
```

Comment:
It contains `share.vimeo.com` which seems to be used for referrals - `https://share.vimeo.com/website_footer`, so I guess we should exclude it.
Probably the same for some other `||refer.` rules, so I'm not sure if it should be added.

Reply:
I'm not sure as well

@AdguardTeam/filters-maintainers could you please test it for a while?